dts/arm/realtek/ec/rts5912.dtsi

```diff
@@ -82,6 +82,132 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			reg = <0x40090000 0x300>;
+
+			/* GPIO0-GPIO15 */
+			gpioa: gpio@40090000 {
+				compatible = "realtek,rts5912-gpio";
+				gpio-controller;
+				#gpio-cells = <2>;
+				reg = <0x40090000 0x40>;
+				ngpios = <16>;
+				interrupts = <	0 0 1 0 2 0 3 0
+						4 0 5 0 6 0 7 0
+						8 0 9 0 10 0 11 0
+						12 0 13 0 14 0 15 0>;
```

Comment:
same for all of these
```
				interrupts = <0 0 1 0 2 0 3 0
					      4 0 5 0 6 0 7 0
```

etc

Reply:
As the sample, we modified the interrupt numbers format.